pkg/quota_management/quota.go

```diff
@@ -0,0 +1,40 @@
+package quota_management
+
+import (
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/shared"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/shared/utils/arrays"
+)
+
+var defaultBillingModel = BillingModel{
+	ID:          "STANDARD",
+	Expiration:  0, // no expiration
+	GracePeriod: 0, // no grace period
+}
+
+var defaultBillingModels = []BillingModel{defaultBillingModel}
+
+type Quota struct {
+	InstanceTypeID string           `yaml:"instance_type_id"`
+	BillingModels  BillingModelList `yaml:"billing_models,omitempty"`
+}
+
+func (quota *Quota) GetBillingModels() BillingModelList {
+	if len(quota.BillingModels) == 0 {
+		ret := defaultBillingModel
+		ret.Allowed = 0 // if 0, defaults to the MaxAllowedInstances
```

Comment:
This seems a noop. ret is created with the value of `defaultBillingModel` and ret.Allowed is set to 0 but that variable is not returned. or used anywhere